servers/src/mining/stratumserver.rs

```diff
@@ -539,6 +544,12 @@ impl StratumServer {
 				LOGGER,
 				"(Server ID: {}) Solution Found for block {} - Yay!!!", self.id, params.height
 			);
+			worker_stats.num_accepted += 1;
+			let block_found = BlockFound {
+				hash: b.hash().to_hex(),
+			};
+			let block_found_json = serde_json::to_string(&block_found).unwrap();
+			return Ok((serde_json::to_value(block_found_json).unwrap(), share_is_block))
```

Comment:
We cant return here since that will skip the final step of logging all the details of the accepted share.